iidm/iidm-modification/src/main/java/com/powsybl/iidm/modification/topology/CreateVoltageLevelTopology.java

```diff
@@ -117,57 +120,76 @@ public void apply(Network network, boolean throwException, ComputationManager co
             }
             return;
         }
-        // Check voltage level is NODE_BREAKER
         TopologyKind topologyKind = voltageLevel.getTopologyKind();
-        if (topologyKind != TopologyKind.NODE_BREAKER) {
-            LOG.error("Voltage Level {} has an unsupported topology {}. Should be {}", voltageLevelId, topologyKind, TopologyKind.NODE_BREAKER);
-            unsupportedVoltageLevelTopologyKind(reporter, voltageLevelId, TopologyKind.NODE_BREAKER, topologyKind);
-            if (throwException) {
-                throw new PowsyblException(String.format("Voltage Level %s has an unsupported topology %s. Should be %s",
-                        voltageLevelId, topologyKind.name(), TopologyKind.NODE_BREAKER.name()));
-            }
-            return;
+        if (topologyKind == TopologyKind.BUS_BREAKER) {
+            // Create buses
```

Comment:
Check here that switch kind is empty (it should be. If not: warning in any case - no exception).